File: YAzyki programmirovaniya/5 zadach' po fizike na 7 yazykakh programmirovaniya/Задачи VB/WindowsApplication1/Form1.vb
Public Class Form1
    Dim v, S, t As Double

    Private Sub button1_Click_1(sender As Object, e As EventArgs) Handles button1.Click
        t = Val(textBox2.Text)
        S = Val(textBox3.Text)

        v = S / t

        textBox1.Text = CStr(v)
    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub

    Private Sub Button1_Click(sender As System.Object, e As System.EventArgs)


    End Sub



End Class

File: study/mdk/scheme/WindowsFormsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace WindowsFormsApplication1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {

        }

        private void button1_Click(object sender, EventArgs e)
        {
            int x, y, z;
            x = Convert.ToInt32(textBox1.Text);
            y = Convert.ToInt32(textBox2.Text);
            z = Convert.ToInt32(textBox3.Text);
            bool x1, y2, z3, all;
            x1 = Convert.ToBoolean(x | y);
            textBox5.Text = Convert.ToString(x1);
            y2 = Convert.ToBoolean(y | z);
            textBox6.Text = Convert.ToString(y2);
            z3 = Convert.ToBoolean(z | x);
            textBox7.Text = Convert.ToString(z3);
            all = (x1 & y2 & z3);
            textBox8.Text = Convert.ToString(all);
            all = !all;
            textBox9.Text = Convert.ToString(all);
            if (all == true) { textBox4.Text = "5v";
                textBox4.BackColor = Color.Yellow;
            }
            else { textBox4.Text = "0v";
                textBox4.BackColor = Color.Gray;
            }
        }
    }
}
